applications/decentralized_oracles.md

```diff
@@ -0,0 +1,135 @@
+# Decentralized Oracles
+
+- **Team Name:** Lucky Team
+- **Payment Address:** 5FsPMucWPWHGaKD6TiHV2fpbd1497m5sgA7drbBtYic6rhsK
+
+## Project Overview :page_facing_up:
+### Overview
+
+We would like to bring decentralized and trustless oracles for ink! smart contracts.  
+Now that phat contracts are live on Khala and Phala Networks, we are able to provide decentralized oracles for any ink! smart contracts deployed on the Polkadot ecosystem.  
+This project will focus on building two oracles:
+ - VFR
+ - Graph Api Oracle
+
+### Project Details
+
+Recently the Phala team released the Lens Api Oracle.  
+https://medium.com/phala-network/lensapi-oracle-supercharge-your-web3-social-app-a413c936df2b
+
+This oracle is built on offchain rollup request-response for EVM smart contracts.  
+https://github.com/Phala-Network/phat-bricks/tree/master/phat/contracts/action_offchain_rollup
+
+Our team has applied to the Phala builder program to develop the offchain rollup request-response for ink! smart contracts.
+
+When this milestone is completed, we would like to use this offchain rollup request-response model to build:
+ - VRF Oracle. It should generate cryptographically secure random numbers for ink! smart contracts
+ - Graph API Oracle that: 
+   - accesses SubQuery/SubSquid via an URI
+   - transform and format the response via the JS engine developed by the Phala team
+   - send the output to the ink! smart contract
+
+For both oracle, we will provide:
+ - the phala offchain rollup for the VRF or access SubQuery/SubSquid
+ - the ink! smart contracts to interact with the offchain rollup
+ - a UI to test and play with these smart contracts. For the Graph API Oracle, we will index the data from the dApps Staking on Astar and return the information about the stakers
+ - the documentation to use these oracles
+
+The smart contracts will be deployed on Shiden and Khala.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- GuiGou is application architect in Web 2 and works more and more in web3.   
+As a technical ambassador for Astar and ambassador for Phala, he promotes the ink! smart contracts via some tutorials (https://polkaverse.com/11143) and replies about the technical questions on ink! smart contracts via Astar's discord.
+- Arno is a software engineer, web developer and ambassador for Polkadot, Phala, Acala and Talisman Sentinel
+
+
+### Team's experience
+
+We developed the Lucky dApp, built on top of dApp Staking in Astar Network.  
+If the user stakes on the Lucky dApp, he will still receive the rewards from the dApp Staking, and furthermore, he will have a chance to win extra rewards. It’s a no-loss lottery!  
```

Comment:
Well I guess it's "no-loss" in the sense that the token amounts don't go down, but there will be inflation related loses, right?

And where are the "extra rewards" coming from?